cmd/config.go

```diff
@@ -42,6 +42,9 @@ type Config struct {
 	DisableEncoderMigrations  bool
 	HypersyncMaxQueueSize     uint32
 
+	// Mempool
+	MempoolBackupTimeMilliseconds uint64
```

Comment:
I would call this an "interval" because it's how often you back it up: `MempoolBackupIntervalMillis`